CRVResponse/src/CrvStepsFromStepPointMCs_module.cc

```diff
@@ -61,6 +61,9 @@ namespace mu2e
 	  Comment("Removes steps of neutral particles"),true};
 	fhicl::Atom<bool> useTotalEDep{ Name("useTotalEDep"),
 	  Comment("Use total energy deposition instead. Visible is a new default"),false};
+	fhicl::Atom<bool> noPostPositionAvailable{ Name("noPostPositionAvailable"),
+	  Comment("No postposition is available for CRY3/4 samples"),false};
+
```

Comment:
I'd like this bundled with the above to be a single 'backwards compatibility' mode.  Also please put comments in the code that this is a deprecated feature and will be eliminated soon.

Reply:
> I'd like this bundled with the above to be a single 'backwards compatibility' mode. Also please put comments in the code that this is a deprecated feature and will be eliminated soon.

We might want to toggle this flags individually for debugging purposes.
I don't have a strong preference though. I can do that if you insist.

Reply:
OK.  I see the comment basically defines them as deprecated.